internal/options/options.go

```diff
@@ -191,3 +209,11 @@ func GetLDOptionsFromEnv() (map[string]string, error) {
 
 	return ldOptions, nil
 }
+
+func GetDebugOptionFromEnv() (bool, error) {
+	debug := os.Getenv("debug")
```

Comment:
i think the convention is for env vars to be all uppercase. also, i'm a fan of making specific env var flags like (like `LDDEBUG`), because there's a high chance that `DEBUG` will collide with some other env var names and trigger debug behavior in some other program.

Reply:
ooh- i think this was an oversight, the variable was supposed to be `LD_DEBUG`. good catch!